ui/src/app/pages/data/data-set/export-dataset-modal.spec.tsx

```diff
@@ -62,16 +70,22 @@ describe('ExportDatasetModal', () => {
       expect(screen.queryByLabelText('Please Wait')).toBeNull();
     });
 
-  const changeNotebookName = (newNotebookName) => {
-    fireEvent.change(findNotebookNameInput(), {
-      target: { value: newNotebookName },
-    });
+  const changeNotebookName = async (newNotebookName) => {
+    await user.type(findNotebookNameInput(), newNotebookName);
   };
 
   const clickExportButton = () => {
     fireEvent.click(findExportButton());
   };
 
+  const clickShowCodePreviewButton = () => {
+    fireEvent.click(findSeeCodePreviewButton());
```

Comment:
I've had some trouble with clicking ... is there a good guide for when to use this vs. `user.click()` vs. `element.click()` ?

Reply:
I think that most of them should be user-events:
https://testing-library.com/docs/dom-testing-library/api-events/

I believe that I started with fireEvents though. I should work on porting these over.

Reply:
I was able to transition them all to user-events.